packages/dialog/src/dialog.css

```diff
@@ -12,17 +12,83 @@ governing permissions and limitations under the License.
 
 @import './spectrum-dialog.css';
 
-::slotted([slot='button']) {
-    margin-left: var(
-        --spectrum-buttongroup-button-gap-x,
-        var(--spectrum-global-dimension-static-size-200)
-    );
-}
-
 .content {
     overflow: hidden;
 }
 
 .content[tabindex] {
     overflow: auto;
 }
+
+::slotted(img[slot='hero']) {
+    width: 100%;
+    height: auto;
+}
+
+/* NEEDS MEDIA QURIE SUPPORT */
```

Comment:
Hrm. I'm not sure why this is here...I'll correct the spelling and spell out why we've got this here 100% for next future me to not be so confused.

Reply:
Removed comment AND the duplicated code!

Reply:
EDIT: This was needed. I expanded the comment so that we know how/when to remove. The functionality in question (#593) should really come into active work soon as it's pretty important...